client/src/feedbacker-components/survey-edit/survey-edit-container.scss

```diff
@@ -25,6 +25,51 @@
   }
 }
 
+.slack-button {
+  align-items: center;
+  display: flex;
+  background: #00c2cc;
+  justify-content: center;
+
+  span {
+    color: white;
+  }
+
+  // Visually centered vertically
+  svg {
+    fill: white;
+    position: relative;
+    height: 17px;
+    width: 17px;
+    top: 1.1px;
+    margin: 3.5px;
+  }
+}
+
+// TODO: these styles to be abstracted to organisms later
+
+.overlay {
```

Comment:
This is can be found as partial in _modal.scss extend from there

Reply:
fixd?